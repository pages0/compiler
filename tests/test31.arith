empty 2::1::[] : int