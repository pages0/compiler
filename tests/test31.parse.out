( empty ( Cons 2 ( Cons 1  [] : int ) ) )